include/oneapi/dpl/pstl/pstl_config.h

```diff
@@ -81,13 +82,13 @@
 #    define _PSTL_PRAGMA_SIMD_REDUCTION(PRM)
 #endif //Enable SIMD
 
-#if (__INTEL_COMPILER)
+#if (__INTEL_LLVM_COMPILER || __INTEL_COMPILER)
 #    define _PSTL_PRAGMA_FORCEINLINE _PSTL_PRAGMA(forceinline)
 #else
 #    define _PSTL_PRAGMA_FORCEINLINE
 #endif
 
-#if (__INTEL_COMPILER >= 1900)
+#if (__INTEL_LLVM_COMPILER >= 20230100 || __INTEL_COMPILER >= 1900)
```

Comment:
Is there a compiler version which doesn't  stratify` (__INTEL_LLVM_COMPILER >= 20230100 || __INTEL_COMPILER >= 1900)`?

Reply:
Sure, there are even among the supported ones. For example, Intel(R) oneAPI DPC++/C++ Compiler 2022.2.1.

Reply:
and?.. at the same tine `__INTEL_COMPILER >= 1900 `is true?

Reply:
DPC++ compilers do not define `__INTEL_COMPILER`.